config/crds/clusterlink.net_imports.yaml

```diff
@@ -36,8 +36,9 @@ spec:
             description: Spec represents the attributes of the imported service.
             properties:
               lbScheme:
```

Comment:
nit: Envoy uses `policy`, not `scheme` for this

Reply:
@zivnevo

Reply:
Back in the old days, we used Policy to encapsulate all the details the user had to set in a given execution of `gwctl` (client name, service name, scheme and default peer). Now that the scheme is set inside an Import CRD, I don't mind renaming it  as `lbPolicy`. I opened Issue #423 .